packages/graphql/README.md

```diff
@@ -6,230 +6,14 @@
 
 This is a [preset for Hops](https://github.com/xing/hops/tree/master#presets) that can be used to set up an `<ApolloProvider />` and enable server-side and client-side support for GraphQL via the Apollo framework. Additionally it also brings an Apollo Server that can be used for mocking to enable faster local development.
 
+This preset is just combining the two presets [`hops-apollo`](https://github.com/xing/hops/blob/master/packages/apollo/README.md#presets) and [`hops-apollo-mock-server`](https://github.com/xing/hops/blob/master/packages/apollo-mock-server/README.md#presets).
+
 ### Installation
 
-_This preset must be used together with the `hops-react` preset._
+_This preset must be used together with the [`hops-react`](<(https://github.com/xing/hops/blob/master/packages/react/README.md#presets)>) preset._
 
 Add this preset and its peer dependencies `graphql-tag` and `react-apollo` to your existing Hops React project:
```

Comment:
This peer-dependency list is incomplete. Maybe we just refer to the peer-dependencies of hops-mock-server & hops-apollo here?!